src/main/java/ladder/domain/Player.java

```diff
@@ -0,0 +1,21 @@
+package ladder.domain;
+
+public class Player {
+    public static final int MAX_NAME_LENGTH = 5;
+    private String name;
```

Comment:
```suggestion
    private static final int MAX_NAME_LENGTH = 5;
    
    private String name;
```

상수화 필드는 개행으로 분리해서 분관해두면 더 좋습니다.